tests/integration/defs/accuracy/test_llm_api_pytorch_multimodal.py

```
@@ -438,3 +438,86 @@ def test_auto_dtype(self):
+            "nvidia/Nemotron-3-Nano-Omni-30B-A3B-Reasoning-NVFP4",
+            f"{llm_models_root()}/NVIDIA-Nemotron-3-Nano-Omni-30B-A3B-Reasoning-NVFP4",
+            KvCacheConfig(
+                free_gpu_memory_fraction=0.75,
+                mamba_ssm_cache_dtype="float32",
+                enable_block_reuse=False,
+                dtype="fp8",
+            ),
+            marks=skip_pre_blackwell,
+            id="nvfp4",
+        ),
+    ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)